force-app/main/default/lwc/tableauViz/__tests__/tableauViz.test.js

```diff
@@ -162,6 +182,67 @@ describe('tableau-viz', () => {
         );
     });
 
+    it('handles multiple record filters', async () => {
+        const element = createElement('c-tableau-viz', {
+            is: TableauViz
+        });
+        element.vizUrl = VIZ_URL;
+        element.filterOnRecordId = false;
+        element.height = 650;
+        element.objectApiName = 'Account';
+        element.recordId = 'mockId';
+        element.sfAdvancedFilter = 'Account.Name';
+        element.tabAdvancedFilter = 'Name';
+        element.addRecordFilter('Account.Owner', 'Owner');
+        element.addRecordFilter('Account.Alias', 'Nickname');
+        document.body.appendChild(element);
+
+        getRecordWireAdapter.emit(mockGetRecordMultipleFilters);
+
+        await flushPromises();
+
+        const vizPlaceholder = element.shadowRoot.querySelector(
+            'div.tabVizPlaceholder'
+        );
+        expect(vizPlaceholder).not.toBeNull();
+        expect(global.tableauMockInstances.length).toBe(1);
+        const instance = global.tableauMockInstances[0];
+        expect(instance.vizToLoad).toBe(
+            `${VIZ_DISPLAY}${vizPlaceholder.offsetWidth}%2C650&Name=SpacelySprockets&Owner=Acme&Nickname=Jimmy`
```

Comment:
How do we know the parameters would be in this order? Sounds like a magic to me. I must have missed something 😄